src/views/htmlcontent/src/css/color-theme.css

```diff
@@ -265,6 +260,7 @@
     background: transparent;
   }
 
+  .vscode-light .shortCut,
```

Comment:
If you're doing this you can just remove the `.vscode` selectors and just have a single selector

```
.shortCut {
    color: var(--color-content);
  }
```